docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3'
 
 services:
   mysql:
-    image: mysql:5.7.22
+    image: mysql:8-oracle
```

Comment:
Our other images have just been using mysql:8.0, but looks like this is basically the same thing currently. So that's fine. 

https://github.com/tl-its-umich-edu/my-learning-analytics/blob/58e61f5afd77319a46152b44af31ffcf551fb742/docker-compose.yml#L6